projects/libprotobuf-mutator/build.sh

```diff
@@ -21,10 +21,13 @@ cp -f $SRC/*.dict $SRC/*.options $OUT/
 mkdir -p build
 pushd build
 rm -rf *
-cmake .. -GNinja -DCMAKE_BUILD_TYPE=Release \
+
+CXXFLAGS="$CXXFLAGS -Wl,-lpthread -Wno-unused-command-line-argument" \
```

Comment:
pthread is needed by probuf library if --stdlib=libc++ specified.
Compiler does add -pthread and looks like it's expected behavior as libc++ user may use no threads  related std:: stuff.
As it's only libprotobuf-mutator issue, and no other oss projects need this, I added -pthread into cmake file there.